ccd-definition/CaseEventToFields/CreateSDO.json

```diff
@@ -57,8 +57,18 @@
     "PageID": "ClaimsTrack",
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Y",
+    "ShowSummaryChangeOption": "N",
     "FieldShowCondition": "claimsTrack = \"fastTrack\""
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CREATE_SDO",
+    "CaseFieldID": "orderType",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderType",
+    "PageDisplayOrder": 3,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "N"
```

Comment:
Will the render logic for this page come in a future PR? 

To my knowledge, this page will only render based on the field value of the first screenshot in the ticket being 'no'.

Reply:
Yes thats right, there will be a follow up change based on Suvarna's work